### PR TITLE
PA-2458 Real Estate Analyst can edit properties again

### PR DESCRIPTION
**Pull request by agalbrai**
https://user-images.githubusercontent.com/33818575/104241827-1a763b80-5413-11eb-9e0d-c688a40dba24.mp4

### CODE REVIEW

File: frontend/src/features/properties/components/forms/subforms/BuildingForm.tsx [hunk 1]
```diff
@@ -252,8 +252,4 @@ const BuildingForm = (props: BuildingProps & FormikProps<any>) => {
   );
 };
 
-export default React.memo(BuildingForm, (prevProps, currentProps) => {
-  const prev = { formikProps: prevProps, field: prevProps.nameSpace };
-  const curr = { formikProps: currentProps, field: currentProps.nameSpace };
-  return formikFieldMemo(prev, curr) && prevProps.disabled !== currentProps.disabled;
-});
```
Comment (agalbrai) [inline frontend/src/features/properties/components/forms/subforms/BuildingForm.tsx:259]: This was stopping the props from changing when you clicked Edit on the Building page, and disabled was not being reset so the fields were uneditable

File: frontend/src/features/properties/components/forms/subforms/EvaluationForm.tsx [hunk 2]
```diff
@@ -167,8 +167,4 @@ const EvaluationForm = <T extends any>(props: EvaluationProps & FormikProps<T>)
   );
 };
 
-export default React.memo(EvaluationForm, (prevProps, currentProps) => {
-  const prev = { formikProps: prevProps, field: prevProps.nameSpace };
-  const curr = { formikProps: currentProps, field: currentProps.nameSpace };
-  return formikFieldMemo(prev, curr);
-});
```
Comment (agalbrai) [inline frontend/src/features/properties/components/forms/subforms/EvaluationForm.tsx:174]: same here

**Review by goemen — APPROVED** [review]
Looks good to me

**Review by Fosol — APPROVED** [review]
Thank you very much!

**Pull request merged**